packages/specs/schema/src/components/default/inheritedClassMapper.ts

```
@@ -12,10 +29,9 @@ export function inheritedClassMapper(obj: any, {target, ...options}: JsonSchemaO
 
   if (stores.length) {
     const schema = stores.reduce((obj, [, store]) => {
-      return deepMerge(obj, execMapper("schema", [store.schema], options));
+      return deepMerge(obj, execMapper("schema", [store.schema], options), {alter: alterMerge});
     }, {});
-
-    obj = deepMerge(schema, obj);
+    obj = deepMerge(schema, obj, {alter: alterMerge});
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)